lib/process-services-cloud/src/lib/task/directives/claim-task-cloud.directive.spec.ts

```diff
@@ -37,6 +37,10 @@ describe('ClaimTaskCloudDirective', () => {
 
         @ViewChild(ClaimTaskCloudDirective, { static: false })
         claimTaskDirective: ClaimTaskCloudDirective;
+
+        onError(error: Error) {
```

Comment:
this method does not do anything except returning what you give it, why it's needed?

Reply:
we are checking the `error`  method is called or not when API failed.